docs/High-Level_Design.md

```diff
@@ -0,0 +1,112 @@
+| Master state data in the right pace | The state data of services and resources will remain mastered within platforms. This will include access control to resources and configuration of infrastructure. These state data will be read from the platforms (Such as TRE, RDP and Condenser) and processes managed within the portal may trigger changes in state within the platform. | TBC | TBC
+| Retention of Quality Data | Where processes within the portal make changes to metadata or resources adequate quality data will be captured and retained in line with the requirements of any downstream system. | TBC | TBC
+
+## Non-functional Requirements
+
+### Data Classification
+
+The portal and underlying data will be limited to storing confidential and public data.
+
+### Availability
+
+TBC
+
+### Integrity
+
+See above principle on quality data.
+
+## Definition of the MVP
+
+
+
+## Conceptual data model
+This model represents an overview of the current understanding of the target main entities referenced in the project stories, with minimal detail. It improves in some respects from the model in the [as-is Portal](#as-is-portals).
+
+```mermaid
+erDiagram
+User
+Study {
+    URL dpoRegistration
+    URL ethics
+    URL[] grantAwards
+}
+Study |o--|| User : "has owner"
+Study |o--|{ User : "has admins"
+Asset {
+    enum requiredTier
+}
+Study ||--o{ Asset : "IG for"
+Asset }o--|| Asset : parent
+Contract {
+    URL *contract
+    boolean hasExternalUsers
+}
+Asset }o--o{ Contract : "used under"
+Project |o--|| User : "has IT admin"
+Study ||--o{ Project : "IG boundary for"
+Project }o--o{ Asset : uses
+Environment {
+    URL *location
+    enum tier
+    User owner
+}
+Environment ||--|{ Project : contains
+```
+Notes on syntax:
+- [Crow's foot notation](https://en.wikipedia.org/wiki/Entity%E2%80%93relationship_model#Crow's_foot_notation)
+- Some attributes use URLs to refer to entities in other systems. The local entity may express some inherited data about the referenced entity which is useful for local business rules, e.g. whether a Contract allows external users.
+
+### Study
+A **Study** is an abstract Information Governance boundary which is interpreted according to context. It may closely correspond to a real-world research study, a whole lab, or a research project or programme.
+
+### Asset
+An **Asset** belongs strictly to one **Study**. If the Study transfers an Asset to another Study, a new Asset is created. This should reference the original Asset through the `parent` relationship, so that provenance can be established. This relationship can carry additional information such as what derivation was used, e.g. anonymisation.
+
+### Project
+The **Project** entity type is expected to be subtyped for specific Environments, including with additional User roles (beyond the default `itAdmin`). E.g.
+  - The ARC TRE Project subtype specifies desktop users, ingressers, etc.
```

Comment:
Do you mean `isAdmin`?

Reply:
thanks, actually it is "IT", but it definitely deserves expansion.
```suggestion
The **Project** entity type is expected to be subtyped for specific Environments, including with additional environment-specific User roles. The diagram suggests a default IT admin, who is the technical point of contact for the project, but note this role is just a suggestion at this stage. Examples of subtype roles would include:
  - The ARC TRE Project subtype specifies desktop users, ingressers, etc.
```

Reply:
fwiw I'm not a fan of `itAdmin`. owner makes more sense to me

Reply:
that's OK, as it says, it's a weak suggestion to illustrate the expected need for extensibility